index.html

```diff
@@ -5,8 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href = "styles/index.css" rel="stylesheet" type="text/css">
 </head>
 <body>
-    
+    <section id ="headers">
```

Comment:
Instead of using a section with an id of headers you should instead use a header tag:

```suggestion
    <header>
```